src/structures/GuildChannel.js

```diff
@@ -67,14 +66,14 @@ class GuildChannel extends Channel {
       this.parentID = data.parent_id;
     }
 
+    /**
+     * A manager of permission overwrites that belong to this channel
+     * @type {PermissionOverwriteManager}
+     */
+    this.permissionOverwrites = new PermissionOverwriteManager(this);
```

Comment:
Why is this being moved outside of the guard?
This will cause all overwrites to be deleted whenever a channel is patched with partial data.
See: https://github.com/discordjs/discord.js/issues/5605

Reply:
That is intended to be moved outside the guard as this is a manager now, and now with the latest commit, I moved it to the constructor.